### PR TITLE
bpo-46917: math.nan is now always available

### PR DESCRIPTION
**Pull request by vstinner**
https://bugs.python.org/issue46917

### CODE REVIEW

**Comment by vstinner** [comment]
cc @mdickinson

File: Doc/whatsnew/3.11.rst [hunk 1]
```diff
@@ -266,6 +267,9 @@ math
   ``inf``. Previously they raised :exc:`ValueError`. (Contributed by Mark
   Dickinson in :issue:`44339`.)
 
+* The :data:`math.nan` value is now available available.
```
Comment (ajoino) [inline Doc/whatsnew/3.11.rst]: "available" duplicated, done on purpose?

File: Misc/NEWS.d/next/Library/2022-03-10-14-47-16.bpo-46917.s19zcy.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+The :data:`math.nan` value is now available available. Patch by Victor
```
Comment (ajoino) [inline Misc/NEWS.d/next/Library/2022-03-10-14-47-16.bpo-46917.s19zcy.rst]: "available" duplicated, done on purpose?

File: Doc/whatsnew/3.11.rst (hunk 1 above)
Comment (hugovk) [reply Doc/whatsnew/3.11.rst]: Same in the PR title and commit message :)

Comment (vstinner) [reply Doc/whatsnew/3.11.rst]: Nope, it's a silly typo. I just created the PR before leaving my train, without reviewing it, sorry :-) It's now fixed.

Comment (ajoino) [reply Doc/whatsnew/3.11.rst]: was thinking that maybe it was done for emphasis :)

**Pull request merged**